README.md

```diff
@@ -12,6 +12,36 @@ Upstream `Code-OSS` is stored using Git [subtree](https://git-scm.com/book/en/v2
  - NodeJS version used by `Code-OSS` (Exact version can be find inside https://github.com/microsoft/vscode/blob/main/remote/.yarnrc with target property)
  - Yarn v1.x
 
+## Running Visual Studio Code - Open Source ("Code - OSS") in a UBI9-based container
```

Comment:
Let's move this section to the `build/dockerfiles/README.md` file. Not to overload the root README.md doc with a lot of technical details.
Later, we will probably extend the `build/dockerfiles/README.md` with more dockerfile-specific docs.